code/game/objects/items/storage/backpack.dm

```diff
@@ -604,3 +604,18 @@
 	name = "police bag"
 	desc = "A large duffel bag for holding extra police gear."
 	slowdown = 0
+
+//VOIDCREW EDIT BEGIN
+/obj/item/storage/backpack/satchel/kepori
```

Comment:
This should be in the voidcrew folder, if the file doesnt already exist, make one (with the same path), and just paste the code there

Reply:
i tried that and was getting errors about it not existing despite being called at creation